lib/components/header/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { Heading, Box, extendTheme } from '@chakra-ui/react';
+
+const theme = extendTheme({
+  components: {
+    Divider: {
```

Comment:
Why did we change the styles for Divider?

Reply:
we accidentally included this but we don't use this so we are removing this!